app/controllers/miq_policy_controller.rb

```diff
@@ -943,7 +943,9 @@ def folder_get_info(folder_node)
     @sb[:folder] = nodeid.nil? ? nodetype.split("-").last : nodeid
     if x_active_tree == :policy_tree
       if nodeid.nil? && ["compliance", "control"].include?(nodetype.split('-').last)
-        @folders = ["Host #{nodetype.split('-').last.titleize}", "Vm #{nodetype.split('-').last.titleize}"]
+        @folders = MiqPolicy::UI_FOLDERS.collect do |model|
```

Comment:
This seems odd to me... MiqPolicy::UI_FOLDERS is `[Host, Vm, ContainerImage].freeze`.  I know why you did this but it feels like we're using `MiqPolicy::UI_FOLDERS` for the purposes of displaying names of each class... It seems like i18n/gettext/message catalog stuff should be used for these "what do I display" purposes.